packages/react/src/utils/useDebounce/useDebounce.ts

```diff
@@ -0,0 +1,19 @@
+import { useRef } from 'react';
+
+/**
+ * Delays `callback` execution until there are no calls in `debounceTime`.
+ * @param callback Callback to execute.
+ * @param debounceTime Time in milliseconds to debounce events.
+ * @returns
```

Comment:
Why this `@returns` here?

Reply:
Ah auto-inserted when I open a JSDocs comment.
Nice catch, thanks.